qemu/tests/cfg/block_resize.cfg

```diff
@@ -4,15 +4,22 @@
     no RHEL.4
     only qcow2 raw
     type = block_resize
-    disk_change_ratio = "1.5 0.9"
+    extend_ratio = 1.5
+    shrink_ratio = 0.9
+    disk_change_ratio = "${extend_ratio} ${shrink_ratio}"
     accept_ratio = 0.005
+    md5_file = md5.dat
     iozone_path = "WIN_UTILS:\Iozone\iozone.exe"
     iozone_option = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
     Linux:
         iozone_option = " -azR -r 64k -n 512M -g 1G -I -i 0 -i 1 -f %s/testfile"
+        dd_cmd = "dd if=/dev/urandom of=%s oflag=direct bs=1M count=10"
     qcow2:
         Host_RHEL.m6, Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2, Host_RHEL.m7.u3, Host_RHEL.m7.u4, Host_RHEL.m7.u5:
-            disk_change_ratio = "1.5"
+            disk_change_ratio = ${extend_ratio}
+    Windows:
+        md5_cmd = "md5sum.exe %s"
+        dd_cmd = "WIN_UTILS:\coreutils\DummyCMD.exe %s 10240000 1"
```

Comment:
I remember that `DummyCMD.exe` is not friendly in some window OS.
Suggest you copy a file from a host which is generated by command `dd`

Reply:
updated it